test/stability/test1/part1.go

```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/intel-go/yanff/flow"
 	"github.com/intel-go/yanff/packet"
 
-	"github.com/intel-go/yanff/test/stability/test1/common"
+	"github.com/intel-go/yanff/test/stability/test1/test1Common"
```

Comment:
What is it?

Reply:
Just renaming of internal test file, so as not to confuse with github.com/intel-go/yanff/common package. It is not good if we have two packages with same names

Reply:
Of course, however not in this commit... If it is not a problem, can you make a separate commit for it?